help-centre-rest/src/main/resources/application.properties

```diff
@@ -13,6 +13,13 @@ search.default.page.size=25
 spring.mvc.throw-exception-if-no-handler-found=true
 spring.resources.add-mappings=true
 
+############################### Contact Form Config #######################################################
+helpcentre.contact.email.host=smtp.ebi.ac.uk
+#helpcentre.contact.email.to=uniprot-feedback@uniprot.org
+helpcentre.contact.email.to=lgonzales@ebi.ac.uk,xwatkins@ebi.ac.uk,luciani@ebi.ac.uk
```

Comment:
I just added this commented out line to keep the production value in the code.

Reply:
By default I added our e-mails to avoid unwanted e-mails being sent to production email. We will received these e-mails if someone is running some local tests